packages/plugin/tests/input-name.spec.ts

```diff
@@ -10,12 +10,7 @@ ruleTester.runGraphQLTests('input-name', rule, {
       options: [{ checkInputType: true }],
     },
     {
-      code: 'type Mutation { SetMessage(input: SetMessageInput): String }',
```

Comment:
[no-identical-tests](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/no-identical-tests.md) rule tell that this test is the same as on line `9`